src/Menu/Menu.component.tsx

```diff
@@ -28,9 +28,9 @@ const StyledMenu = styled('nav')<MenuProps>`
   width: 100%;
   min-width: 250px;
 
-  ${({ bg = 'primary.base' }) =>
+  ${({ background = 'primary.base' }) =>
```

Comment:
Can this be:

```
  ${({ background: backgroundColor = 'primary.base' }) =>
```

or can be just refactor that prop to `backgroundColor`?

Reply:
I think we'd have to make a decision on how to handle background's in general. I almost made this like `Hero`, which accepts any shorthand for a background. For example, it could handle a gradient whereas `Menu` can't because it is explicitly looking for `background-color` rather than `background`.

I'm against changing the prop to `backgroundColor` unless we standardize the naming across all components. Instances where only the color is changed should be `backgroundColor` whereas instances where it could be an image, gradient, color, etc should be `background`. I lean more towards the latter because it allows for greater flexibility.

Reply:
I have updated this based on our offline conversation